src/pyscipopt/conshdlr.pxi

```diff
@@ -452,3 +460,19 @@ cdef SCIP_RETCODE PyConsGetdivebdchgs (SCIP* scip, SCIP_CONSHDLR* conshdlr, SCIP
     success[0] = False
     infeasible[0] = False
     return SCIP_OKAY
+
+cdef SCIP_RETCODE PyConsGetPermSymGraph (SCIP* scip, SCIP_CONSHDLR* conshdlr, SCIP_CONS* cons, SYM_GRAPH* graph,
+                                         SCIP_Bool* success) noexcept with gil:
+    # TODO
+    PyConshdlr = getPyConshdlr(conshdlr)
+    PyConshdlr.consgetpermsymgraph()
+    success[0] = False
+    return SCIP_OKAY
+
+cdef SCIP_RETCODE PyConsGetSignedPermSymGraph (SCIP* scip, SCIP_CONSHDLR* conshdlr, SCIP_CONS* cons, SYM_GRAPH* graph,
+                                               SCIP_Bool* success) noexcept with gil:
+    # TODO
+    PyConshdlr = getPyConshdlr(conshdlr)
+    PyConshdlr.consgetsignedpermsymgraph()
+    success[0] = False
+    return SCIP_OKAY
```

Comment:
@Opt-Mucca Not important, but I don't understand why this code is not triggered by the tests